go.mod

```diff
@@ -181,4 +160,36 @@ require (
 	gotest.tools/v3 v3.0.3 // indirect
 )
 
+require (
+	github.com/benbjohnson/clock v1.3.0 // indirect
+	github.com/btcsuite/btcd/chaincfg/chainhash v1.0.1 // indirect
+	github.com/cheekybits/genny v1.0.0 // indirect
+	github.com/containerd/cgroups v0.0.0-20201119153540-4cbc285b3327 // indirect
```

Comment:
Please move these in the existing `require` block above

Also running `go mod tidy` may help removing deps